nebula/ui/components/MZTabNavigation.qml

```diff
@@ -47,8 +53,12 @@ Item {
                 id: btn
                 objectName: tabButtonId
                 height: bar.contentHeight
-                Accessible.name: MZI18n[tabLabelStringId]
+                // Qt doesn't correctly provide the selected state for a tab on Windows, so include the selected state in the accessibility name.
```

Comment:
Do we know if there is a bug filed with them for this?

Reply:
https://bugreports.qt.io/browse/QTBUG-120174 has been opened.